v1/k8s/pods/verify-pod-termination.yaml

```diff
@@ -0,0 +1,14 @@
+name: Verify Pod Termination
```

Comment:
Again: what is the use case? 
and how can I specify which pod (regex) should be terminated? 
Maybe for both namespaces and pods we should have a "verify-x-state" and give as input the desired state (terminated, running and maybe others)